test/integration/scheduler_perf/dra.go

```diff
@@ -363,3 +371,278 @@ claims:
 		tCtx.Fatalf("Could not allocate claim %d out of %d", i, len(claims))
 	}
 }
+
+type updateDeviceConditionsOp struct {
```

Comment:
Please add documentation for the new operations:
- what do they do?
- what parameters do they have?

Reply:
Can we update these to explain it in a little more detail? It would be helpful to understand how the ResourceClaims to be updated are identified and how the process is configured with the parameters defined on this op.